cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -68,7 +71,33 @@ def test_appliance_scsi_control_vmware():
         initialEstimate: 1/4h
     #TODO: yet to test this, once done, I will add steps. Test was not written by me originally.
     """
-    pass
+    url = ("{}/{}/{}/cfme-vsphere-paravirtual-{}-1.x86_64.vsphere.ova"
```

Comment:
please use keying for such customization
```
url = "{baseurl}/{series}/{ver}/cfme-vsphere-paravirtual-{ver}-1.x86_64.vsphere.ova".format(
baseurl=conf.cfme_data['basic_info']['cfme_images_url']['baseurl'],
series=appliance.version.series(),
ver=appliance.version
) 
```

Reply:
Makes sense.